packages/Discretization/test/tstMeshGenerator.cpp

```diff
@@ -10,158 +10,146 @@
 #include "MeshGenerator.hpp"
 #include <Teuchos_UnitTestHarness.hpp>
 
+// The out and success parameters come from the Teuchos unit testing macro
+// expansion.
+std::tuple<std::vector<std::vector<Coordinate>>, std::vector<unsigned int>>
+read_input_file( std::string const &filename, bool &success,
+                 Teuchos::FancyOStream &out )
+{
+    std::ifstream file( filename );
+    TEST_ASSERT( file.is_open() );
```

Comment:
Use one of the DTK macro here rather than `TEST_ASSERT()` and remove `success` and `out` from the parameters.